extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -153,32 +97,52 @@ export class PerFolderServerInstance implements IServerInstance {
 		await this.startInternal();
 	}
 
+	public spawn(command: string, args?: ReadonlyArray<string>, options?: SpawnOptions): ChildProcess {
```

Comment:
Why not just remove this completely and call spawn directly from whatever is calling this?